src/services/ChatService.js

```diff
@@ -206,6 +210,65 @@ const fetchChannel = async ChannelSid => {
   }
 };
 
+const createMessagesForChannel = async (channel, data) => {
+  const { messages } = MessagesUtil;
+  // eslint-disable-next-line no-restricted-syntax
+  for await (const message of messages[channel.friendlyName]) {
+    const formattedMessage = MessagesUtil.getMessage(message, data);
+    const newMessage = {
+      body: formattedMessage,
+      attributes: JSON.stringify({ context: 'casual' }),
+    };
+    await createMessage(newMessage, channel.sid);
+  }
+};
+
+/**
+ * Creates the initial channels for the new user
+ * @param {*} user
+ */
+const createUserChannels = async user => {
```

Comment:
Lets rename this too. createInitialChannels 

I like naming 😬

Reply:
or createNewUserChannels

Reply:
My general naming convention is you should know exactly what the function does by its name.

Reply:
Also done

Reply:
> My general naming convention is you should know exactly what the function does by its name.

You are right, names needed a clear and more accurate name